docs/src/user-docs/guides-mcp-server/mcp-cursor.md

```
@@ -0,0 +1,22 @@
+# Connecting Cursor
+
+This guide explains how to connect Cursor to the CLP MCP server over HTTP.
+
+## Steps
+
+1. Follow the [official Cursor guide](https://cursor.com/docs/context/mcp) to add a Remote MCP Server using `mcp.json`.
+2. Add the following `clp-mcp-server` config to the `mcpServers` object and save the file.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)